recipes/aws-noaa-solar-irradiance-daily/recipe.py

```diff
@@ -0,0 +1,35 @@
+from os.path import join
+import s3fs
+from pangeo_forge_recipes.patterns import pattern_from_file_sequence
+from pangeo_forge_recipes.recipes.reference_hdf_zarr import HDFReferenceRecipe
+
+s3_bucket = 'noaa-cdr-total-solar-irradiance-pds'
+s3_files_path = 'data/daily/'
+url_base = f's3://{s3_bucket}/{s3_files_path}'
+file_list = []
+fs = s3fs.S3FileSystem(anon=True)
+
+
+def is_nc(x):
+    return x.endswith('.nc')
```

Comment:
there are 3 preliminary files. we should either filter these out if there are corresponding, non preliminary files at the same time and location. or include them if there are not corresponding files.

Reply:
yes, I've filtered those preliminary files!!